internal/store/cronjob.go

```diff
@@ -96,11 +96,15 @@ func cronJobMetricFamilies(allowAnnotationsList, allowLabelsList []string) []gen
 			basemetrics.STABLE,
 			"",
 			wrapCronJobFunc(func(j *batchv1.CronJob) *metric.Family {
+				timeZone := "local"
+				if j.Spec.TimeZone != nil {
+					timeZone = *j.Spec.TimeZone
+				}
 				return &metric.Family{
 					Metrics: []*metric.Metric{
 						{
-							LabelKeys:   []string{"schedule", "concurrency_policy"},
-							LabelValues: []string{j.Spec.Schedule, string(j.Spec.ConcurrencyPolicy)},
+							LabelKeys:   []string{"schedule", "concurrency_policy", "timezone"},
```

Comment:
wouldn't it be better in its own metric rather than in the info metric?

Reply:
There's a 1:1 relationship, why would you prefer a dedicated metric for it?

Reply:
@dgrisonnet any feedback here?

Reply:
`_info` is not a very intuitive suffix which is why I don't really like these metrics where we just end up dumping a bunch of labels that aren't really related between one another.

I don't know what was the historical reasoning for putting labels into these metrics, but I can also see some dedicated metrics for values part of objects spec: https://github.com/kubernetes/kube-state-metrics/blob/569e820c65ecfad6c158d5b5eb1ac4fd94cb1b80/internal/store/pod.go#L1212-L1231

To me, info should be limited to the default labels. Any other information that we want to expose should be in its own dedicated metric. What is your opinion on that?

Reply:
I agree on having a separate metric for anything that does not have a 1:n relationship within the object.  For 1:1 relationships, additional time series make it difficult to correlate between multiple labels vs. simply extracting the ones you need: 
`max(resource_info) by (label1, label2, label3)`.
Trying to do the same with multiple metrics can get very annoying and more error-prone.

To me, the _info metric should describe all single-value keys of the object that are static and non-numeric.

Reply:
Hmm, there seems to be a naming discrepancy between per-property gauges and per-object info metrics, so there's no clear pattern we can adhere to.

That being said, accumulating all cardinally-bound fields as label-sets into a per-object `_info` metric opens the room for metrics with a large number of labels. The metric itself can grow to become unintentionally cardinal and dropping unwanted labels will entail relabelling efforts. Splitting this into per-field metric will limit the cardinality and allow for a more granular control.

Users should be able to add the timezone label to any exported metrics using a `foo_metric * on(job) group_left(timezone) kube_cronjob_timezone`.

Reply:
I tried to define it here:
`the _info metric should describe all single-value keys of the object that are static over its lifetime and non-numeric`

Reply:
I meant a documentation/standard of sorts in the repository that enforces this. Not sure if there is one.

Reply:
I can add that in a follow-up PR.

Reply:
I'm currently working on a tool that will generate ideal metric names but it will take some time.

Reply:
I believe the tool's definition on an incorrect or correct metric representation for a nested object needs to weigh in the point Damien raised above.

Basically, are we in consensus on what is the standard approach for representing a nested value (since more granularity leads to more joins, but more abstractions lead to more cardinal label-sets (which is comparatively more harmful))?